tests/integration/test_http.py

```diff
@@ -92,3 +92,26 @@ def test_timeouts(self):
             correct_exception = False
 
         self.assertTrue(correct_exception)
+
+    def test_read_timeouts(self):
```

Comment:
I think these tests belong in the unit tests instead. Here's a patch:

```diff
commit 869c3129ca04cadc6945ca489ef2232f96f9a63b (HEAD -> bhargavrpatel-master-add-read-timeouts)
Author: Blade Barringer <blade.barringer@getbraintree.com>
Date:   Mon Jun 24 20:12:08 2019 +0000

    Move integration tests to unit tests

diff --git a/tests/integration/test_http.py b/tests/integration/test_http.py
index 5c415b9..b87218a 100644
--- a/tests/integration/test_http.py
+++ b/tests/integration/test_http.py
@@ -92,26 +92,3 @@ class TestHttp(unittest.TestCase):
             correct_exception = False

         self.assertTrue(correct_exception)
-
-    def test_read_timeouts(self):
-        """It should handle a request read timeout."""
-        try:
-            raise requests.exceptions.ReadTimeout()
-        except braintree.exceptions.http.timeout_error.ReadTimeoutError:
-            correct_exception = True
-        except Exception:
-            correct_exception = False
-
-        self.assertTrue(correct_exception)
-
-
-    def test_connection_timeouts(self):
-        """It should handle a request connection timeout."""
-        try:
-            raise requests.exceptions.ConnectTimeout()
-        except braintree.exceptions.http.timeout_error.ConnectTimeoutError:
-            correct_exception = True
-        except Exception:
-            correct_exception = False
-
-        self.assertTrue(correct_exception)
diff --git a/tests/unit/test_http.py b/tests/unit/test_http.py
index 213ff20..661d9b0 100644
--- a/tests/unit/test_http.py
+++ b/tests/unit/test_http.py
@@ -1,6 +1,7 @@
 import traceback

 from tests.test_helper import *
+from braintree.exceptions.http.timeout_error import *
 from braintree.attribute_getter import AttributeGetter

 class TestHttp(unittest.TestCase):
@@ -108,3 +109,19 @@ class TestHttp(unittest.TestCase):
                 "wrap_http_exceptions": False})

         return Http(config, "fake_environment")
+
+    @raises(ReadTimeoutError)
+    def test_raise_read_timeout_error(self):
+        def test_http_do_strategy(http_verb, path, headers, request_body):
+            return (200, "")
+
+        http = self.setup_http_strategy(test_http_do_strategy)
+        http.handle_exception(requests.exceptions.ReadTimeout())
+
+    @raises(ConnectTimeoutError)
+    def test_raise_read_timeout_error(self):
+        def test_http_do_strategy(http_verb, path, headers, request_body):
+            return (200, "")
+
+        http = self.setup_http_strategy(test_http_do_strategy)
+        http.handle_exception(requests.exceptions.ConnectTimeout())
```

Reply:
Moved it to the recommended test file; thanks.